pandas/tests/tools/test_to_datetime.py

```diff
@@ -1540,7 +1556,10 @@ def test_to_datetime_malformed_raise(self):
         ts_strings = ["200622-12-31", "111111-24-11"]
         with pytest.raises(
             ValueError,
-            match=r"^hour must be in 0\.\.23: 111111-24-11, at position 1$",
+            match=(
+                r"^offset must be a timedelta strictly between "
+                r"-timedelta\(hours=24\) and timedelta\(hours=24\)., at position 0$"
+            ),
```

Comment:
this is a fun one. on 1.5.x, if you only parse the first input, you get the "offset must be a timedelta strictly..." message, but not if you include the second one
```python
In [6]: to_datetime(["200622-12-31"])

ValueError: offset must be a timedelta strictly between -timedelta(hours=24) and timedelta(hours=24).

In [7]: to_datetime(["200622-12-31", "111111-24-11"])

ParserError: hour must be in 0..23: 111111-24-11
```

Now, you'll get the same message in both cases